modules/notes/script_component.hpp

```diff
@@ -0,0 +1,2 @@
+#define COMPONENT GRAD_leaveNotes
```

Comment:
Does it matter that the component name differs from the `tag = ` decalaration in CfgFunctions? I always set them to be the same, but I dunno if that matters or not?

Reply:
should all be COMPONENT NOTES